openruth.module

```diff
@@ -433,13 +433,20 @@ function _openruth_user_status($creds = NULL, $reset = FALSE) {
  */
 function openruth_seek_reserved_item($faust_number) {
   global $user;
+
+  if (!function_exists('openruth_reservation_list')) {
+    include_once(drupal_get_path('module', 'openruth') . '/includes/openruth.reservation.inc');
+  }
```

Comment:
The function "openruth_reservation_list" is located in the inc file, but the auto load in drupal 7 is for classes and interfaces only. The file may not be loaded when the code above is executed, as the files section in the info file don't have any effect.

So this must basically be the same as the provider do in: https://github.com/ding2/ding_provider/blob/master/ding_provider.module#L535 to load these inc files from a provider.

That said it seams that the function (openruth_seek_reserved_item) is only used from inside "openruth.loan.inc". So maybe it should be move into "openruth.reservation.inc", which then would require "openruth.loan.inc" to load "openruth.reservation.inc"?

Any better solutions @pjohans or @kasperg ?